motr/m0crate/tests/test1.yaml

```diff
@@ -26,37 +26,36 @@
 
 CrateConfig_Sections: [MOTR_CONFIG, WORKLOAD_SPEC]
 MOTR_CONFIG:
-  MOTR_LOCAL_ADDR: 10.0.2.15@tcp:12345:34:123
-  MOTR_HA_ADDR: 10.0.2.15@tcp:12345:34:101
-  PROF: <0x7000000000000001:1>
-  LAYOUT_ID: 1
-  BLOCK_SIZE: 1048576
-  IS_OOSTORE: 1
-  IS_READ_VERIFY: 0
-  TM_RECV_QUEUE_MIN_LEN: 2
-  MAX_RPC_MSG_SIZE: 131072
-  PROCESS_FID: <0x7200000000000000:0>
-  IDX_SERVICE_ID: 1
-  CASS_CLUSTER_EP: "127.0.0.1"
-  CASS_KEYSPACE: "motr_index_keyspace"
-  CASS_MAX_COL_FAMILY_NUM: 1
+    MOTR_LOCAL_ADDR: 192.168.52.53@tcp:12345:4:1
+    MOTR_HA_ADDR: 192.168.52.53@tcp:12345:1:1
+    PROF: <0x7000000000000001:0x3a>
+    LAYOUT_ID: 1
+    IS_OOSTORE: 1
+    IS_READ_VERIFY: 0
+    TM_RECV_QUEUE_MIN_LEN: 2
+    M0_MAX_RPC_MSG_SIZE: 131072
+    PROCESS_FID: <0x7200000000000001:0x1b>
+    IDX_SERVICE_ID: 1
+    CASS_CLUSTER_EP: "127.0.0.1"
+    CASS_KEYSPACE: "motr_index_keyspace"
+    CASS_MAX_COL_FAMILY_NUM: 1
 
 WORKLOAD_SPEC:
-  WORKLOAD_TYPE: 0
-  WORKLOAD_SEED: tstamp
-  NUM_KVP: 8
-  NXRECORDS: default # int or default
-  RECORD_SIZE: 32 # int [units] or random
-  MAX_RSIZE: 1M # int [units]
-  OP_COUNT: 32 # int [units] or unlimited = (2 ** 31 - 1) / (128 * NUM_KVP)
-  EXEC_TIME: unlimited # int (seconds) or unlimited
-  WARMUP_PUT_CNT: 0 # int (ops) or all
-  WARMUP_DEL_RATIO: 0 # int (ops / ratio)
-  KEY_PREFIX: random # int
-  KEY_ORDER: ordered # ordered or random
-  INDEX_FID: <7800000000000001:0> # fid
-  PUT: 100 # int
-  DEL: 0 # int
-  GET: 0 # int
-  NEXT: 0 # int
-  LOG_LEVEL: 4 # err(0), warn(1), info(2), trace(3), debug(4)
+    WORKLOAD_TYPE: 0
+    WORKLOAD_SEED: tstamp
+    NUM_KVP: 8
+    NXRECORDS: default # int or default
+    RECORD_SIZE: 32 # int [units] or random
```

Comment:
since we are updating these files, lets update to use latest params as key size and value size and just deprecate RECORD_SIZE

Reply:
same for other sample yaml files.

Reply:
yes, we can remove the "RECORD_SIZE" from these sample yaml config files.

Reply:
Removed RECORD_SIZE and MAX_RSIZE from sample yaml config files.